modules/compliance-profiles.adoc

```diff
@@ -45,113 +65,143 @@ description: |-
-metadata:
-  annotations:
-    compliance.openshift.io/product: redhat_enterprise_linux_coreos_4
-    compliance.openshift.io/product-type: Node
-    creationTimestamp: "2020-09-07T11:42:51Z"
+  id: xccdf_org.ssgproject.content_profile_e8
+  kind: Profile
+  metadata:
+    annotations:
+      compliance.openshift.io/image-digest: pb-rhcos426smj
+      compliance.openshift.io/product: redhat_enterprise_linux_coreos_4
+      compliance.openshift.io/product-type: Node
+    creationTimestamp: "2022-02-08T21:17:03Z"
     generation: 1
-  labels:
-    compliance.openshift.io/profile-bundle: rhcos4
-  name: rhcos4-e8
-  namespace: openshift-compliance
-rules:
-- rhcos4-accounts-no-uid-except-zero
-- rhcos4-audit-rules-dac-modification-chmod
-- rhcos4-audit-rules-dac-modification-chown
-- rhcos4-audit-rules-execution-chcon
-- rhcos4-audit-rules-execution-restorecon
-- rhcos4-audit-rules-execution-semanage
-- rhcos4-audit-rules-execution-setfiles
-- rhcos4-audit-rules-execution-setsebool
-- rhcos4-audit-rules-execution-seunshare
-- rhcos4-audit-rules-kernel-module-loading
-- rhcos4-audit-rules-login-events
-- rhcos4-audit-rules-login-events-faillock
-- rhcos4-audit-rules-login-events-lastlog
-- rhcos4-audit-rules-login-events-tallylog
-- rhcos4-audit-rules-networkconfig-modification
-- rhcos4-audit-rules-sysadmin-actions
-- rhcos4-audit-rules-time-adjtimex
-- rhcos4-audit-rules-time-clock-settime
-- rhcos4-audit-rules-time-settimeofday
-- rhcos4-audit-rules-time-stime
-- rhcos4-audit-rules-time-watch-localtime
-- rhcos4-audit-rules-usergroup-modification
-- rhcos4-auditd-data-retention-flush
-- rhcos4-auditd-freq
-- rhcos4-auditd-local-events
-- rhcos4-auditd-log-format
-- rhcos4-auditd-name-format
-- rhcos4-auditd-write-logs
-- rhcos4-configure-crypto-policy
-- rhcos4-configure-ssh-crypto-policy
-- rhcos4-no-empty-passwords
-- rhcos4-selinux-policytype
-- rhcos4-selinux-state
-- rhcos4-service-auditd-enabled
-- rhcos4-sshd-disable-empty-passwords
-- rhcos4-sshd-disable-gssapi-auth
-- rhcos4-sshd-disable-rhosts
-- rhcos4-sshd-disable-root-login
-- rhcos4-sshd-disable-user-known-hosts
-- rhcos4-sshd-do-not-permit-user-env
-- rhcos4-sshd-enable-strictmodes
-- rhcos4-sshd-print-last-log
-- rhcos4-sshd-set-loglevel-info
-- rhcos4-sshd-use-priv-separation
-- rhcos4-sysctl-kernel-dmesg-restrict
-- rhcos4-sysctl-kernel-kexec-load-disabled
-- rhcos4-sysctl-kernel-kptr-restrict
-- rhcos4-sysctl-kernel-randomize-va-space
-- rhcos4-sysctl-kernel-unprivileged-bpf-disabled
-- rhcos4-sysctl-kernel-yama-ptrace-scope
-- rhcos4-sysctl-net-core-bpf-jit-harden
-title: Australian Cyber Security Centre (ACSC) Essential Eight
+    labels:
+      compliance.openshift.io/profile-bundle: rhcos4
+    name: rhcos4-e8
+    namespace: openshift-compliance
+    ownerReferences:
+    - apiVersion: compliance.openshift.io/v1alpha1
+      blockOwnerDeletion: true
+      controller: true
+      kind: ProfileBundle
+      name: rhcos4
+      uid: 6b6090b6-2dce-4f33-bbf9-e9ddd4c66f56
```

Comment:
@pdhamdhe - Is the uid: line sensitive? Should it be removed from the example? Is there any other information that should be removed from the updated example output?

Reply:
@kmccarron-rh Hi, it's not sensitive but we can remove it to shorten the output along with that  remove `resourceVersion`, `creationTimestamp` & `generation` etc

Reply:
Done! Thanks for the info